UMAprotocol/voter-dapp-v2
# fix: z index in dropdown

The z index of the items in the dropdown component was not high enough to beat the z index of the panel. This caused the dropdown items to not show in the pagination dropdown for the vote history.

* add max z index to dropdown items so that they are always on top.

To test:

go to the vote history panel and check that the pagination dropdown (the one where you select results per page) works as expected.
